bats/tests/registry/creds.bats

```diff
@@ -103,6 +108,22 @@ EOF
     wait_for_container_engine
 }
 
+@test 'verify credential is set correctly' {
+    verify_default_credStore
+}
+
+verify_default_credStore() {
+    local CREDHELPER_NAME="$(basename "$CRED_HELPER" .ext | sed s/docker-credential-//)"
+
+    assert_file_exist "$CONFIG_FILE"
+    if command -v jq; then
+        assert_equal "$CREDHELPER_NAME" "$(jq -r .credsStore "$CONFIG_FILE")"
```

Comment:
Write tests in the simplest form possible. Use `run` and `assert_success`, and then the `assert_output` functions to test the actual output. This makes it easier to see exactly what failed, if the test doesn't pass.

```suggestion
        run jq -r .credsStore "$DOCKER_CONFIG_FILE"
        assert_success
        assert_output "$CREDHELPER_NAME"
```

Reply:
Makes sense